invenio_openaire/loaders.py

```diff
@@ -277,6 +321,66 @@ def iter_grants(self, as_json=True):
         self._disconnect()
 
 
+class LocalJSONOAIRELoader(BaseOAIRELoader):
+    """Local OpenAIRE dataset loader."""
+
+    def __init__(self, source=None, **kwargs):
+        """Init the loader for local database."""
+        super(LocalJSONOAIRELoader, self).__init__(
+            source,
+            **kwargs)
+
+    def extract_grants(self, source):
+        """Extract grants within a json lines file."""
+        parsed_grants = []
+        with gzip.open(source, 'r') as json_file:
+            json_list = list(json_file)
+
+        for json_grant in json_list:
+            parsed_grants.append(json.loads(json_grant))
+
+        return parsed_grants
+
+    def extract_fundings(self, grants):
+        """Extract fundings from a grant dumps."""
+        parsed_fundings = []
+
+        for grant in grants:
+            funding = grant["funding"]
+            if funding not in parsed_fundings:
+                parsed_fundings.append(funding)
+
+        return parsed_fundings
+
+    def extract_funders(self, fundings):
+        """Extract funders from fundings."""
+        parsed_funders = []
+        unparsed_fundings = []
+
+        for funding in fundings:
+            try:
+                funder = funding[0]['shortName']
+                if funder not in parsed_funders:
+                    parsed_funders.append(funder)
+
+            except:
```

Comment:
```suggestion
            except Exception:
```

In order to not break `ctrl-C`